app/models/course.rb

```diff
@@ -11,6 +11,8 @@ class Course < ApplicationRecord
 
   delegate :short_title, to: :path, prefix: true
 
+  scope :badges, -> { Course.where(show_on_homepage: true) }
```

Comment:
Sorry  @ChargrilledChook, I missed this on the first run through. Can we add a spec for this scope?

Reply:
Sure thing :ok_hand: